tests/tck/features/match/PathExpr.feature

```diff
@@ -50,22 +50,22 @@ Feature: Basic match
       """
       MATCH (v:player) WITH (v)-[v]-() AS p RETURN p
       """
-    Then a SemanticError should be raised at runtime: Alias `v' should be Edge, but got type 'Node'
```

Comment:
Why change this?

Reply:
The previous error message is a bit ambiguous.